src/components/QuickExitButton.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
 
 export default function EndChat({ channelSid, token }: Props) {
   // Serverless call to end chat
-  const handleEndChat = async () => {
+  const fetchEndChat = async () => {
```

Comment:
Personally prefer the old name, its not really fetching something but asking for a change in the system.